nucypher/characters/chaotic.py

```diff
@@ -129,7 +129,7 @@ class Felix(Character, NucypherTokenActor):
     research and the development of production-ready nucypher dApps.
     """
 
-    _default_crypto_powerups = [SigningPower, BlockchainPower]
+    _default_crypto_powerups = [SigningPower]
```

Comment:
Why does Felix lose his BlockchainPower?

Reply:
At the moment, I cannot determine a way for `BlockchainPower` to be consumed as a default powerup, it's constructed and consumed in the `__init__`  of the character.